launch_testing/test/check_env.py

```diff
@@ -0,0 +1,4 @@
+if __name__ == '__main__':
+    import os
+    assert(os.environ.get('testenv2', 'Not Set') == 'Not Set')
```

Comment:
Surprisingly it raised errors as expected with the extra brackets.
Removed in https://github.com/ros2/launch/pull/38/commits/f49d6b681fa3c475976730e42f04b1d1a34de890